projects/aws-containers/hello-eks-anywhere/README.md

```diff
@@ -0,0 +1,5 @@
+## **Hello EKS Anywhere image**
+![Version](https://img.shields.io/badge/version-v0.1.1-blue)
+![Build Status](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiZThzcTBqV1o0ZVN6QXVIem96alF3dVZidnVwT1hyT3BYS0FoRU9WdlJnY0NySmFwb0dXWGdsMVBndzNxRjZWWXJlYWdnRi9xc3VZZmpKdzRHZnJEQUtJPSIsIml2UGFyYW1ldGVyU3BlYyI6ImFvOU5wRk9LRThzWE5nL00iLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
+
+Hello EKS Anywhere for testing EKS Anywhere Packages. Validate deployment, configuration and secrets.
```

Comment:
Can we add an updating section similar to https://github.com/aws/eks-anywhere-build-tooling/tree/main/projects/fluxcd/flux2#updating?

Reply:
I can, it's not nearly as complex as flux, but I'll add that.

Reply:
Added in 66d8764